core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1135,6 +1136,71 @@ object TestUtils extends Logging {
   }
 
   /**
+   * Verifies that this ACL is the secure one. The
+   * values are based on the constants used in the
+   * ZooKeeper code base.
```

Comment:
Much better, maybe we can remove the comment "The values are based on constants.." since it seems redundant.